### DISCUSSION TITLE
Addressing the limitations for large message volumes

### DISCUSSION THREAD

**Discussion by thedewpoint** [rabbitmq/rabbitmq-delayed-message-exchange#193] [General]
Hi, I was curious if the limitations for message volumes apply to the entire node, or if it is distributed to each of the delayed exchanges? I'm wondering if I am able to handle a higher volume by distributing the volume on multiple exchanges, maybe by sharding using a hash. Do all of these exchanges share the same disk? Or would this help alleviate some of the issues called out in the documentation?

**Comment by thedewpoint** [comment]
@michaelklishin since I'm not sure if a discussion triggers a notification or not

**Comment by michaelklishin** [comment]
All exchanges are backed by the same schema database table.
